acrobat/scripts/scripts.js

```diff
@@ -184,8 +184,10 @@ function decoratePromotion() {
   const { default: lanaLogging } = await import('./dcLana.js');
 
   // Setup CSP
-  const { default: ContentSecurityPolicy } = await import('./contentSecurityPolicy/csp.js');
-  ContentSecurityPolicy();
+  if (window.location.pathname.indexOf('online') > 0) {
```

Comment:
```suggestion
  if (window.location.pathname.includes('online/') {
```
Should do.. but again, just be aware that will be true any URL includes `online/`. 
I would be happier if we make this working with metadata instead. 

Something like..
```
const frictionlessMeta = getMetadata('frictionless');
  if (frictionlessMeta === 'on') {
```

Reply:
At least that's how milo core does when URL based condition is required...

Reply:
i hear you! let's do that then!